packages/utils.parser/spec/parserBehaviors.ts

```diff
@@ -461,17 +465,6 @@ describe('unary operations', function () {
     assert.equal(bindings.neg(), 3)
   })
 
-  it.skip('negates an expression eg !(a || b)'
```

Comment:
re-add

Reply:
Done in this PR. @mcselle removed it because `it.skip` is Mocha and not well defined in the type system. I've re-added and commented out those skip cases.